src/GovUk.Education.ExploreEducationStatistics.Admin/Mappings/MappingProfiles.cs

```diff
@@ -222,6 +224,46 @@ public MappingProfiles()
                 .ForMember(dest => dest.Type,
                     m => m.MapFrom(dataSetVersion =>
                         dataSetVersion.VersionType));
+
+            CreateMap<DataSetUpload, DataSetUploadViewModel>()
+                .ForMember(dest => dest.Status,
+                    m => m.MapFrom(upload =>
+                        GetDataSetUploadStatus(upload.ScreenerResult)))
+                .ForMember(dest => dest.DataFileSize,
+                    m => m.MapFrom(upload =>
+                        FileExtensions.DisplaySize(upload.DataFileSizeInBytes)))
+                .ForMember(dest => dest.MetaFileSize,
+                    m => m.MapFrom(upload =>
+                        FileExtensions.DisplaySize(upload.MetaFileSizeInBytes)));
+
+            CreateMap<DataSetScreenerResponse, ScreenerResultViewModel>();
+
+            CreateMap<DataScreenerTestResult, ScreenerTestResultViewModel>()
+                .ForMember(dest => dest.Result,
+                    m => m.MapFrom(upload => upload.Result.ToString()));
+
+            CreateMap<DataSetUpload, DataSetScreenerRequest>()
+                .BeforeMap((s, d) => d.StorageContainerName = "releases-temp");
```

Comment:
Can the string value `"releases-temp"` be referenced from a constant somewhere rather than hard-coded here?